examples/queries/examples/query_deployed_bytecode.rs

```diff
@@ -0,0 +1,30 @@
+//! Example of querying deployed bytecode of a contract on Ethereum network.
+
+use alloy_network::Ethereum;
+use alloy_primitives::address;
+use alloy_provider::{HttpProvider, Provider};
+use alloy_rpc_client::RpcClient;
+use alloy_rpc_types::{BlockId, BlockNumberOrTag};
+use alloy_transport_http::Http;
+use eyre::Result;
+use reqwest::Client;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let provider = init();
+
+    // Get bytecode of USDC-ETH Uniswap V3 pool
+    let pool_address = address!("88e6A0c2dDD26FEEb64F039a2c41296FcB3f5640");
+
+    let bytecode =
+        provider.get_code_at(pool_address, BlockId::Number(BlockNumberOrTag::Latest)).await?;
+
+    println!("Bytecode: {:?}", bytecode);
+
+    Ok(())
+}
+
+fn init() -> HttpProvider<Ethereum> {
+    let http = Http::<Client>::new("https://eth.llamarpc.com".parse().unwrap());
+    HttpProvider::new(RpcClient::new(http, true))
```

Comment:
tbh, I find this a but much.
it shouldn't require 3 steps to configure an Http provider

a few suggestions:

https://github.com/alloy-rs/alloy/blob/main/crates/rpc-client/src/client.rs#L50-L50

there should be `RpcClient::local`

https://github.com/alloy-rs/alloy/blob/main/crates/transport-http/src/lib.rs#L39-L39

there should be a function for reqwest::Client

needs discussion:
there should be a shortcut for reqwest HTTP

https://github.com/alloy-rs/alloy/blob/main/crates/provider/src/provider.rs#L68-L68

Reply:
I agree. Ref: https://github.com/alloy-rs/alloy/issues/318

Reply:
https://github.com/alloy-rs/alloy/blob/main/crates/rpc-client/src/client.rs#L50-L50 these can be combined into a helper args struct

```rust
HttpTransportArgs<T> {
 transport: T
 is_local
}
```

so that we can accept `impl Into<HttpTransportArgs<T>>` here, I think this could work

Reply:
Please see examples in `main` for latest usage of shorthands

Related: https://github.com/alloy-rs/alloy/issues/340 + https://github.com/alloy-rs/alloy/issues/387